framework/auth/views.py

```diff
@@ -143,8 +144,8 @@ def auth_login(auth, **kwargs):
         if (campaign == 'institution' and settings.ENABLE_INSTITUTIONS) or campaign != 'institution':
             data['campaign'] = campaign
     data['login_url'] = cas.get_login_url(redirect_url, auto=True)
-
     data['sign_up'] = request.args.get('sign_up', False)
+    data['existing_user'] = request.args.get('existing_user')
```

Comment:
You should probably define a default: `request.args.get('existing_user', False)`